boards/nxp/imx95_evk/board.c

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/cache.h>
+
+void board_early_init_hook(void)
+{
+	sys_cache_data_enable();
+	sys_cache_instr_enable();
```

Comment:
Should we enable cache in board init, or soc init?

Reply:
Can enable it at soc level. Wasn't sure the NETC stuff needed cache management, hence this is why I restricted it to the ddr variant.

Reply:
I would suggest to enable cache in soc. It will also need MPU region configuration in the future, although it is working fine for your case now.
If we meet any IP driver issue with cache enabling in the future, I think the right way should be trying to fix.
Thanks.